sitestats/sitestats-impl/src/java/org/sakaiproject/sitestats/impl/StatsUpdateManagerImpl.java

```diff
@@ -78,8 +79,7 @@
  */
 @Slf4j
 public class StatsUpdateManagerImpl extends HibernateDaoSupport implements Runnable, StatsUpdateManager, Observer, StatsUpdateManagerMXBean {
-	private final static String				PRESENCE_SUFFIX						= "-presence";
-	private final static int				PRESENCE_SUFFIX_LENGTH				= PRESENCE_SUFFIX.length();
+	private final static int				PRESENCE_SUFFIX_LENGTH				= PresenceService.PRESENCE_SUFFIX.length();
```

Comment:
I'd just inline this as string length is very fast and if it's a performance issue the JIT will probably inline it.

Reply:
Agreed. I just left it as it was so there was less to review :) I'll change it.